scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -53,130 +39,12 @@ export async function sphinxHtmlToMarkdown(options: {
   const meta: PythonObjectMeta = {};
   const isReleaseNotes = url.endsWith("release_notes.html") ? true : false;
 
-  const $page = load(html);
-  const main = $page(`[role='main']`);
-  const $main = $page(main);
-
-  // remove html extensions in relative links
-  $main.find("a").each((_, link) => {
-    const $link = $page(link);
-    const href = $link.attr("href");
-    if (href && !href.startsWith("http")) {
-      $link.attr("href", href.replaceAll(".html", ""));
-    }
-  });
-
-  if (isReleaseNotes && releaseNotesTitle) {
-    // Replace heading with custom heading
-    $page("h1").html(releaseNotesTitle);
-  }
-
-  $main
-    .find("img")
-    .toArray()
-    .forEach((el) => {
-      const $img = $page(el);
-
-      const imageUrl = new URL($img.attr("src")!, url);
-      const src = imageUrl.toString();
-
-      const filename = last(src.split("/"));
-      const dest = `${imageDestination}/${filename}`;
-
-      $img.attr("src", dest);
-
-      if (isReleaseNotes) {
-        // Release notes links should point to the current version
-        $img.attr("src", dest.replace(/[0-9].*\//, ""));
-      }
-
-      images.push({ src, dest: dest });
-    });
-
-  // remove permalink links
-  $main.find('a[title="Permalink to this headline"]').remove();
-  $main.find('a[title="Permalink to this heading"]').remove();
-  $main.find('a[title="Permalink to this definition"]').remove();
-  $main.find('a[title="Link to this heading"]').remove();
-  $main.find('a[title="Link to this definition"]').remove();
-
-  // remove download source code
-  $main.find("p > a.reference.download.internal").closest("p").remove();
-
-  // handle tabs, use heading for the summary and remove the blockquote
-  $main.find(".sd-summary-title").each((_, quote) => {
-    const $quote = $page(quote);
-    $quote.replaceWith(`<h3>${$quote.html()}</h3>`);
-  });
-
-  $main.find(".sd-card-body blockquote").each((_, quote) => {
-    const $quote = $page(quote);
-    $quote.replaceWith($quote.children());
-  });
+  const $ = load(html);
+  const main = $(`[role='main']`);
+  const $main = $(main);
```

Comment:
Do you happen to know what the difference is? Either way, it's good to keep the code the same as before like you are doing. Makes this PR easier to review and less risky.

Reply:
I wasn't sure, but I think it's the same thing, at least with all the tests I did. I removed the second one and renamed the first to `$main`.